local/config/vufind/SolrAlternative.ini

```diff
@@ -0,0 +1,694 @@
+; This section contains global settings affecting search behavior.
```

Comment:
Is there a reason you committed a file in local/config/vufind? Normally the core code should only ship with configs in config/vufind.

Reply:
no; indeed I committed it accidently